utils/install-llvm.sh

```diff
@@ -8,38 +8,38 @@ re_number="^[0-9]+$"
 re_llvm_release="^llvmorg-[0-9]+\.[0-9]+\.[0-9]+$"
 
 num_cores=${1}
-build_dir="${2}"
-dest_dir="${3}"
-llvm_release="${4}"
+dest_dir="${2}"
+llvm_release="${3}"
 
-if [ "$#" -ne 4 ] || ! [[ "$num_cores" =~ ${re_number} ]] || ! [ -d "${build_dir}" ] || ! [[ "${llvm_release}" =~ ${re_llvm_release} ]]; then
-	echo "usage: <prog> <# cores> <build dir> <install dir> <LLVM release (e.g. 'llvmorg-9.0.0')>" >&2
+if [ "$#" -ne 3 ] || ! [[ "$num_cores" =~ ${re_number} ]] || ! [[ "${llvm_release}" =~ ${re_llvm_release} ]]; then
+    echo "usage: <prog> <# cores> <install dir> <LLVM release (e.g. 'llvmorg-9.0.0')>" >&2
 	exit 1
 fi
 
-
 if [ -x ${dest_dir}/bin/llvm-config ]; then
    version=`${dest_dir}/bin/llvm-config --version`
    echo "Found LLVM ${version} already installed at ${dest_dir}."
    exit 0;
 fi
 
 echo "Getting the LLVM source code..."
-if [ ! -d "${build_dir}/llvm-project" ]; then
+tmp_dir=/tmp/${llvm_release}
```

Comment:
```suggestion
tmp_dir=$(mktemp -d -t phasar-llvm-${llvm_release}-XXXXXXXXXX)
```
This makes it a bit more robust in cases where the system defines a different `tmp` or in case of clean-up failures.

Reply:
Good point.